### ISSUE TITLE
The docs link in the readme is 404

### ISSUE THREAD

**Issue by LilithHafner** [barrettp/Astrometry.jl#3]

**Comment by barrettp** [comment]
Reset repository permissions to read-write.

**Comment by abhro** [comment]
Hi, the docs link is still giving a 404. Any idea as to why?

**Comment by barrettp** [comment]
Yes. I haven't had time to fix it. It is currently not high on priority list. I may have time in the next week or so when I update and register the FITS.jl package.
